### ISSUE TITLE
Checkout: Incorrect currency shown at checkout when paying with GPay without adjustment for currency conversion

### ISSUE THREAD

**Issue by pauljacobson** [Automattic/wp-calypso#65082]
### Quick summary

Our user reported that when they attempted to renew their WordPress.com Personal Plan and domain subscription from their mobile device using Google Pay, the checkout page displays the renewal cost in USD without converting the value of the transaction from INR, like this:

![Screenshot_2022-06-29-14-29-39-108_com android chrome](https://user-images.githubusercontent.com/2912202/176406576-56b78cb6-4149-415b-9bae-2a1a4cc5eafd.jpg)

Instead, the renewal cost should be either reflected at this value in INR, or converted into the correct USD value.

The user has selected Google Pay for payment.

### Steps to reproduce

1. Add a renewal for the WordPress.com Personal Plan and the domain to the cart on a mobile device;
2. Observe that the renewal cost has the correct value for a INR, although the currency is reflected in USD.

### What you expected to happen

The renewal cost should be presented in INR like this:

![Capture_2022-06-29 at 12 37 38](https://user-images.githubusercontent.com/2912202/176407049-163d654b-c7a6-4b91-9e6a-727abaf643f7.jpeg)

### What actually happened

The renewal cost is incorrectly presented in USD without an appropriate currency conversion.

### Context

Customer report in 5335014-zen

### Simple, Atomic or both?

Simple

### Theme-specific issue?

Theme doesn't appear to be relevant

### Browser, operating system and other notes

The device seems to be an Android device. I have requested more details from the user.

### Reproducibility

Once

### Severity

One

### Available workarounds?

Yes, easy to implement

### Workaround details

I'm unaware of a workaround for this on the mobile device. The desktop browser reflects the correct pricing.

**Comment by syhussaini** [comment]
I just played into this ticket and asked them to ensure they are not using any VPN when trying to make the payment to isolate the issue. see 5335014-zen for further notes.

**Comment by cuemarie** [comment]
>Steps to reproduce
>>Add a renewal for the WordPress.com Personal Plan and the domain to the cart on a mobile device;
>>Observe that the renewal cost has the correct value for a INR, although the currency is reflected in USD.

@pauljacobson to clarify, were you able to replicate this issue, following these steps? Or is the example above specific to just this user?

---

Leaving open for now as I cannot complete the steps to replicate this. Next PQA, here are the full steps to replicate as far as I can tell:

**1. You need to have Google Pay methods added to a Chrome account, which is also connected to a Mobile Chrome Browser**
2. Open SA for the test account and change the currency to INR
3. In mobile Chrome browser (necessary for Google Pay), log in to the test account on WordPress.com and add a purchase renewal to your cart.
4. Proceed to the checkout page to see if the total shows up in the proper currency, or shows the USD, as above.

I don't use Google Pay, and though I have Google Pay cards set up and Google Pay enabled on my browsers, still couldn't get that pay option up in WordPress.com. Without this step, I can't confirm if this bug is reproducable or not.

**Comment by pauljacobson** [comment]
I haven't reproduced this issue on my side because I'd need to be signed into the user's account in the app on my device.

**Comment by cuemarie** [comment]
@pauljacobson Gotcha, thanks for the quick reply! We'll see if someone in PQAs who has better luck with GPay can reproduce this. So far it's a tough one to replicate- hopefully we get more details from the user as well that might lend more insight. 👍

**Comment by aisajib** [comment]
📌 SCRUBBING
- Tested on Calypso ✅ 

📌 FINDINGS/SCREENSHOTS/VIDEO

I was able to replicate this on a test account. Here's the eCommerce plan on the checkout page in INR:

<img width="653" alt="Markup 2022-07-10 at 21 45 32" src="https://user-images.githubusercontent.com/2541844/178152057-9dd327b0-e0c6-4de5-b933-7c1ae207f1a2.png">

When the Google Pay prompt shows up, I see this: 

<img width="1145" alt="Markup 2022-07-10 at 21 46 01" src="https://user-images.githubusercontent.com/2541844/178152079-762f5947-f9e2-4d55-957b-027a7ea32d5a.png">

Not sure if Google would have ended up charging in USD instead of in INR, but I think it would have ended up charging in USD. 

📌 ACTIONS

- Marked as Triaged for Quality Squad review

**Comment by syhussaini** [comment]
I was able to reproduce this with the following steps:

1. Open WordPress.com on a mobile browser (Chrome)
2. Proceed to checkout with any of the active purchases.
3. The currency initially shows in INR, but upon selecting GPay as the payment method, it switches to USD

_See the image below for reference_
![image](https://d.pr/i/Qrr2Ia+)
_Please [click here](https://d.pr/i/Qrr2Ia) if you are unable to see the image above_

But when I try to use another payment method, it works normally without changing the currency.
I have placed ticket 5335014-zen on hold for now.
